src/OpenLoco/src/Map/TileClearance.h

```diff
@@ -36,12 +36,14 @@ namespace OpenLoco::World::TileClearance
     bool applyClearAtStandardHeight(const World::Pos2& pos, uint8_t baseZ, uint8_t clearZ, const QuarterTile& qt, std::function<ClearFuncResult(TileElement& el)> clearFunc);
     bool canConstructAt(const World::Pos2& pos, uint8_t baseZ, uint8_t clearZ, const QuarterTile& qt);
 
+    using RemovedBuildings = sfl::small_set<World::Pos3, 128, LessThanPos3>;
```

Comment:
I went and looked up vanilla and it used a max of 100 but think 128 is also good.